proto_proxy.c

```diff
@@ -2605,6 +2648,16 @@ static void proxy_process_command(conn *c, char *command, size_t cmdlen, bool mu
         return;
     }
 
+    // Count requests handled by proxy vs local.
+    // Also batch the counts down this far so we can lock once for the active
+    // counter instead of twice.
+    struct proxy_int_stats *istats = c->thread->proxy_int_stats;
+    WSTAT_L(c->thread);
+    istats->counters[pr.command]++;
+    c->thread->stats.proxy_conn_requests++;
+    c->thread->stats.proxy_req_active++;
```

Comment:
We're increasing `proxy_req_active` here, but we increase it again in `proxy_run_coroutine()`. Wouldn't that count the same command twice?

Reply:
That's a DECR in `proxy_run_coroutine()`